modules/template/src/main/twirl/scaladex/view/project/artifact.scala.html

```diff
@@ -62,6 +62,7 @@ <h3>
               @if(artifact.resolver.isEmpty) {
                 @info("Files") { <a href="@artifact.mavenReference.repoUrl">View all</a> }
               }
+              @artifact.fullScalaVersion.map{ version => @info("Scala version") { @version.toString }}
```

Comment:
I think we need to be more specific:

```suggestion
              @artifact.fullScalaVersion.map{ version => @info("Full Scala version") { @version.toString }}
```